net/p910nd/files/p910nd.hotplug

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+user=
+
+. /lib/functions.sh
+
+config_load p910nd
+config_get user p910nd user lp
+
+case "$ACTION" in
+        add)
+		[ -n "${DEVNAME}" ] && [ "${DEVNAME##usb/lp*}" = "" ] && {
+			chmod 660 /dev/"$DEVNAME"
+			chgrp "$user" /dev/"$DEVNAME"
```

Comment:
Likely this should be chown.

That said, maybe we should just used fixed user/group name.  No need to vary it by a configurable option.

Reply:
Actually group works well...and leaves the owner as root, which I think is better, but if you really want it to be lp user I can do that.

Reply:
It's just that it does not look right with `chgrp "$user" xxx`.  We expect `$user` to be a user-configurable system username.  The code will fail if the name is not also a group name.

Reply:
I think assigning fixed group ownership to device files is a practice worth following.  E.g. `tty`, `disk`, `kvm`, etc.

Reply:
Ah, ok, yeah I got that wrong.  I could do "$(id -gn $user)" but that's messy.  I'll just do fixed group ownership.

Reply:
@yousong @pkerling Pushed changes that combine this and @luizluca comments -- will resolve conversation once I've tested them

Reply:
@yousong @pkerling @luizluca Latest iteration successfully tested...please take a look!